apps/fabric-examples/src/pages/ButtonPage/examples/Button.Primary.Example.tsx

```diff
@@ -16,7 +16,8 @@ export class ButtonPrimaryExample extends React.Component<IButtonProps, {}> {
         <PrimaryButton
           data-automation-id='test'
           disabled={ disabled }
-        >Create account</PrimaryButton>
+          text="Create account"
+        >Text Overrides a Child String</PrimaryButton>
```

Comment:
This is a good test case to have, but do we want it to be in the example? I would think in general people should be using the `text` prop **or** children and not both.